packages/back-end/src/controllers/features.ts

```diff
@@ -614,6 +615,9 @@ export async function putFeature(
 
   const updatedFeature = await updateFeature(org, feature, updates);
 
+  // If there are new tags to add
+  await addTagsDiff(org.id, feature.tags || [], updates.tags || []);
+
```

Comment:
should we also include a migration script to handle previously-entered feature tags?

Reply:
@bryce-fitzsimons Mmmmm.... good call. I'm definitely not opposed to it.

I'm guessing this isn't affecting a lot of orgs, so I could also see an argument for just letting it ride.

Thinking out loud, the script would just need to go through each org, and look at the tags at the org level, and compare those tags to the tags of each feature, and if there is a tag on a feature that doesn't exist at the org level, add it.

In the past I think we've ran the migration separately - I'm not sure we're set up to automatically run such things. I don't have write access to our prod DB, so I'll probably need to sync with Jeremy to run the migration.

Reply:
we'd probably want it to run for self-hosted as well, so I'd think maybe it could be a runonce on app start for the current org. I'm not sure how we do those currently though...

Reply:
@bryce-fitzsimons I'm going to bring this to standup as a discussion item, and get this merged in for now.